port/mbedtls/wpa_supp_els_pkc_mbedtls_config.h

```diff
@@ -3995,16 +3995,77 @@ void *pvPortCalloc(size_t num, size_t size); /*Calloc for HEAP3.*/
  */
 
 #define MBEDTLS_ALLOW_PRIVATE_ACCESS
+#define MBEDTLS_CTR_DRBG_ENABLED
+#define MBEDTLS_CIPHER_AES_ENABLED
+#define MBEDTLS_CIPHER_MODE_CBC_ENABLED
+#define MBEDTLS_CIPHER_PADDING_PKCS7
+#define PSA_CRYPTO_DRIVER_ELS_PKC
```

Comment:
can you please share some references to ELS-PKC and also elaborate commit log about the nature of error and how the fix fixes it? (typo in commit log, els_pck -> ELS PKC)

Reply:
Hi @krish2718 , what do you mean that some references to ELS-PKC?

Reply:
update the commit message

Reply:
I am not familiar with ELS-PKC, so, was wondering if you can provide some context about this and how/why this is used in WPA supplicant? I only see below usage in Zephyr repo, but that still doesn't give enough context.

```
$ ag ELS_PKC ../
../modules/hal/nxp/mcux/mcux-sdk/drivers/common/fsl_common.h
201:    kStatusGroup_ELS_PKC        = 161, /*!< Group number for ELS PKC status codes. */

../modules/lib/hostap/port/mbedtls/wpa_supp_els_pkc_mbedtls_config.h
58:#define PSA_CRYPTO_DRIVER_ELS_PKC

../modules/tee/tf-m/trusted-firmware-m/platform/ext/target/nxp/common/Native_Driver/drivers/fsl_common.h
198:    kStatusGroup_ELS_PKC        = 161, /*!< Group number for ELS PKC status codes. */
```

Reply:
Hi @krish2718 @jukkar @rlubos  please review again

Reply:
> Hi @krish2718 the mention of els_pkc is not appropriate in the commit message, so I remove it. actually the Els-pkc is proprietary component of nxp, which provides HW acceleration for psa-apis

Thanks. So, this header file isn't used upstream?

Reply:
this header file isn't used upstream

Reply:
Thanks, then IMHO we should remove this as it's only NXP related. @jukkar WDYT?

Reply:
Yes, I agree

Reply:
@MaochenWang1 can you please update this PR to remove this file instead?

Reply:
yes, will update

Reply:
Hi @jukkar @krish2718  remove as your comment